addons/website/views/snippets/s_banner_categories.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_categories" name="Banner Categories">
+    <section
+        data-name="Banner Categories"
+        class="s_banner_categories o_cc o_cc3 oe_img_bg o_bg_img_center pb56"
+        data-oe-shape-data="{'shape':'web_editor/Connections/20','colors':{'c5':'rgb(255, 255, 255)'},'showOnMobile':false}"
+        style="background-image: url('/web/image/website.s_banner_categories_default_image_1'); background-position: 50% 75%;"
+        >
+        <div class="o_we_bg_filter bg-black-50"/>
+        <div
+            class="o_we_shape o_web_editor_Connections_20"
+            style="background-image: url('/web_editor/shape/web_editor/Connections/20.svg?c5=rgb%28255%2C255%2C255%29');"
+            />
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11" style="gap: 15px;">
+                <div
+                    data-name="Intro"
+                    class="o_grid_item o_colored_level g-height-6 g-col-lg-8 order-0"
+                    style="z-index: 1; --grid-item-padding-y: 70px; grid-area: 2 / 3 / 8 / 11;"
+                    >
+                    <h1 style="text-align: center;">Explore Our New Collection</h1>
+                    <p class="lead" style="text-align: center;">
+                        Each item in our collection is grouped by category to make your browsing experience effortless. Take a look and dive into the different worlds we’ve curated for you.
+                        <br/>
+                        <br/>
+                    </p>
+                    <p style="text-align: center;">
+                        <a
+                            t-att-href="cta_btn_href"
+                            class="btn btn-lg btn-secondary o_translate_inline bg-white border-0"
+                            >
+                            <t t-esc="cta_btn_text">Buy Now</t>
+                        </a>
+                    </p>
+                </div>
+                <div
+                    class="o_grid_item o_colored_level g-col-lg-4 g-height-6 col-lg-4 o_cc o_cc5 oe_img_bg o_bg_img_center col-10 offset-1 offset-lg-0 order-1"
+                    style="grid-area: 8 / 1 / 14 / 5; z-index: 1;background-image: url('/web/image/website.s_banner_categories_default_image_2');align-content: end; --grid-item-padding-y: 30px; --grid-item-padding-x: 30px;"
```

Comment:
This align-content is not something we can do or modify with the editor.

That said, maybe you can add an option in the cards to allow the user to align the content vertically. You can check the `s_quadrant` snippet to see how it's done 😉

<img width="280" alt="Screenshot 2025-05-16 at 09 31 58" src="https://github.com/user-attachments/assets/688375c8-ae81-43f1-8e99-1adf0e48869e" />

```suggestion
                    style="grid-area: 8 / 1 / 14 / 5; z-index: 1;background-image: url('/web/image/website.s_banner_categories_default_image_2'); --grid-item-padding-y: 30px; --grid-item-padding-x: 30px;"
```